src/HDWallet.h

```diff
@@ -59,7 +59,7 @@ class HDWallet {
     virtual ~HDWallet();
 
     /// Returns the private key at the given derivation path.
-    PrivateKey getKey(const DerivationPath& derivationPath) const;
+    PrivateKey getKey(enum TWCoinType coin, const DerivationPath& derivationPath) const;
 
```

Comment:
it's a little bit not intuitive to provide coin as slip44 and derivation path at the same time, is there a way to init derivationPath with a coin type?

Reply:
we need coin to get the curve type

Reply:
but `derivationPath` has a coin right? I would personally keep it as simple as possible, not a big fan of slip048 standard, should we just follow bip44 for all coins?

Reply:
Yes, we can close this